### DISCUSSION TITLE
📣 An announcement about the future of the AWS Pricing Plugin for Google Sheets

### DISCUSSION THREAD

**Discussion by reganbrianpatrick** [mheffner/aws-pricing#49] [Announcements]
[Macroscope](https://macroscope.io/), in collaboration with [@mheffner](https://github.com/mheffner), is excited to announce that we are inheriting Mike's Google Sheets plugin to continue the great work he started. In the near future we will be getting the plugin back into a working state with the latest AWS pricing data. In addition we're actively reviewing all existing issues and will be responding after they have been prioritized.

We're excited to bring AWS pricing data back to your Google Sheets and we'll be announcing some expanded features soon. In the meantime feel free to raise issues, and to collaborate on the project please reach out to us at [hello@macroscope.io](mailto:hello@macroscope.io) or comment on the discussion here. We'd love to hear from you!

**Comment by ACBatconsumeraffairs** [comment]
Great news, Macroscope and Mike! Thank you for committing time to make this great tool work again!

**Reply by reganbrianpatrick** [reply]
Absolutely.. work is in progress!
